components/hab/src/command/pkg/binlink.rs

```diff
@@ -120,7 +120,7 @@ mod test {
         let ident = fake_bin_pkg_install("acme/cooltools", tools, rootfs.path());
         let dst_path = Path::new("/opt/bin");
 
-        let rootfs_src_dir = hcore::fs::pkg_install_path(&ident, None::<&Path>).join("bin");
```

Comment:
There we are computing where the symlink will point to. We want to see an entry like: `/hab/pkgs/.../bin`, that is without any `fs_root_path` set. Otherwise, with the change below, the symlink target will be `/my/tmp/dir/fs/root/hab/pkgs/.../bin` which is outside the `FS_ROOT`.